src/sensors/simulator_bridge.cpp

```diff
@@ -0,0 +1,92 @@
+#include "ros/ros.h"
+#include "std_msgs/Float32.h"
+#include "geometry_msgs/Quaternion.h"
+#include "geometry_msgs/Vector3.h"
+#include "gazebo_msgs/ModelState.h"
+#include "gazebo_msgs/ModelStates.h"
+
+ros::Publisher position_pub;
+ros::Publisher orientation_pub;
+ros::Publisher depth_pub;
+//ros::Publisher lin_accel_pub;
+
+
+// ModelStates msg consists of a name, a pose (position and orientation),
+// and a twist (linear and angular velocity) for each object in
+// the simulator
+// Currently it publishes the position, orientation, and depth
+// of the sub
+// TODO: Eventually this should be generalized to publish a list
+// (defined in params) of objects as well as calulate depth
+void modelStatesCallback(const gazebo_msgs::ModelStates& msg)
+{
+    geometry_msgs::Vector3 position_msg;
+    geometry_msgs::Quaternion orientation_msg;
+    geometry_msgs::Vector3 lin_accel_msg;
```

Comment:
You commented out the publisher for the `lin_accel` shouldn't this also be commented out?